.github/workflows/PRTestJDK11.yml

```diff
@@ -0,0 +1,29 @@
+name: "PR build-jdk-openj9 jdk 11"
+on: # rebuild any PRs and main branch changes
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-16.04, macos-10.15]
+    steps:
+    - uses: sophia-guo/build-jdk-openj9@v1
```

Comment:
This part should be delivered to the openj9 repo as well.

Reply:
See my comment in issue #9570 - my thought was to have this action live as a standalone repo (like openj9-systemtest or openj9-website) as it may also be used by those who may to verify changes to openj9-omr, openjdk-tests, or the extensions repos.

<img width="774" alt="Screen Shot 2020-05-19 at 10 50 14 AM" src="https://user-images.githubusercontent.com/2836948/82342178-501b6f80-99bf-11ea-9ee6-a7638db9ce48.png">

Reply:
The configure step is missing some arguments to be comparable with OpenJ9 builds.
Currently: 
```
if (parseInt(version) > 8)  bootjdkConfigure = `--with-boot-jdk=${workDir}/bootjdk`
bash configure --with-freemarker-jar=${workDir}/freemarker.jar ${bootjdkConfigure}`
```

Linux should also use:
all: `--enable-jitserver --openssl-version=1.1.1g --with-openssl=fetched --enable-cuda --with-cuda=<path>`
8: '--disable-zip-debug-info'

Mac:
all: `--openssl-version=1.1.1g --with-openssl=fetched --enable-openssl-bundling`
8:
`--with-xcode-path=.../Xcode4/Xcode.app --with-openj9-cc=.../clang --with-openj9-cxx=.../clang++ --with-openj9-developer-dir=.../Developer`

Windows:
all: `--openssl-version=1.1.1g --with-openssl=<path> --enable-openssl-bundling --enable-cuda -with-cuda=<path>`
8: '--disable-zip-debug-info --with-freetype-include=.../freetype-2.5.3/include --with-freetype-lib=.../freetype-2.5.3/lib64'
11+: `--with-toolchain-version=2017`

Reply:
Updated with Linux and Mac. For Windows will update when enabling. 
https://github.com/sophia-guo/build-jdk-openj9/pull/10

Reply:
`--with-xcode-path=.../Xcode4/Xcode.app --with-openj9-cc=.../clang --with-openj9-cxx=.../clang++ --with-openj9-developer-dir=.../Developer MACOSX_DEPLOYMENT_TARGET=10.9.0 SDKPATH=.../MacOSX10.8.sdk` wasn't meant to be taken literally. The `...` needs to be replaced with a full path. For example, below is the actual commands that OpenJ9 uses, but the paths need to be correct for the machine used to build. Also MACOSX_DEPLOYMENT_TARGET and SDKPATH are environment variables, not configure options. Since JDK8 for Mac needs to build on a older OS version 10.11, probably it can't be built using a github action.

`--with-xcode-path=/Users/jenkins/Xcode4/Xcode.app --with-openj9-cc=/Users/jenkins/Xcode7/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang --with-openj9-cxx=/Users/jenkins/Xcode7/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ --with-openj9-developer-dir=/Users/jenkins/Xcode7/Xcode.app/Contents/Developer`

`SDKPATH=/Users/jenkins/Xcode4/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.8.sdk`

Reply:
macos-10.15 is the only available github-hosted Mac environment for now. And looks like if there will be more Mac versions it will be the newer version. That is saying if we are going to use github-hosted environment there will be no chance to build jdk8 on Mac. Maybe leave those code as comment as TODO.  When there is workable environment we can fix the correct path?

Reply:
Sounds good.

Reply:
Updated. https://github.com/sophia-guo/build-jdk-openj9/pull/12